pkgs/development/python-modules/cvxpy/default.nix

```diff
@@ -45,6 +46,7 @@ buildPythonPackage rec {
     scipy
     scs
     setuptools
+    pybind11
```

Comment:
Does this have to be propagated?

Reply:
I'm not entirely sure.

Reply:
Well `pybind11` is mostly used in the build phase, so unless cvxpy is building more native extensions at runtime, it doesn't need it. That's also the reason I asked about it in relation to the `torch` expression

Reply:
Speaking of which, setuptools is in the wrong list as well

Reply:
Torch does in fact need it.
cvxpy compiled and tested fine with moving setuptools and pybind11 to nativeBuildInputs so I'll just change the commit